server/src/pages/shot/view.js

```diff
@@ -539,8 +539,8 @@ class Body extends React.Component {
       this.props.controller.changeShotExpiration(this.props.shot, INDEFINITE);
     } else {
       sendEvent("unset-favorite", "navbar");
-      const TWO_WEEKS_IN_MS = 1209600000;
-      this.props.controller.changeShotExpiration(this.props.shot, Date.now() + TWO_WEEKS_IN_MS);
+      const TWO_WEEKS_IN_SECONDS = 1209600;
+      this.props.controller.changeShotExpiration(this.props.shot, Date.now() / 1000 + TWO_WEEKS_IN_SECONDS);
```

Comment:
@chenba Are you sure it shoule be in seconds looking at code in controller, expire time is in milliseconds 
https://github.com/mozilla-services/screenshots/blob/fxa-integration/server/src/pages/shot/controller.js#L106

https://github.com/mozilla-services/screenshots/blob/fxa-integration/server/src/pages/shot/model.js#L37

Reply:
Right, which was why I changed  it to ms when you pointed out the unit mismatch in the review of the PR for #4688.  The real bug is that the value should be the number of ms from now, not a timestamp in ms.  I'll update this PR.  Thanks!